ios/forward/forward.go

```diff
@@ -16,28 +16,54 @@ type iosproxy struct {
 	deviceConn ios.DeviceConnectionInterface
 }
 
+type ConnListener struct {
+	listener net.Listener
+	quit     chan bool
```

Comment:
This is a single state channel. Bool has 2 states. I recently learned it is possible to define single state channels as

```
quit chan struct{}
```

or

```
testFinishedChannel chan interface{}
```